lib/tapioca/compilers/dsl/active_record_relations.rb

```diff
@@ -260,7 +266,9 @@ def create_classes_and_includes
             end
 
             create_relation_class
+            create_relation_where_chain_class
```

Comment:
Could we run these as part of `create_relation_class` and `create_association_relation_class` respectively? I feel like it will keep this top level flow much easier to read.

Reply:
Yeah sure thing